### PR TITLE
Rvl 1417

### LINKED ISSUE #1244: End Fund Kenya - Create Tablet Accountability form for MDA
JIRA LINK: https://smartregister.atlassian.net/browse/RVL-1417 
This form should be accessed from the Other forms view.

The form fields and logic are on the tablet accountability form sheet on the data dictionary [here](https://docs.google.com/spreadsheets/d/1J5IsHWSznKEdT82bxGY6Yzfwi43su34mkX0w8bkAeRU/edit#gid=1083089419).

- [x] Create Form
- [x] Wire it up to Other forms section

### PR DESCRIPTION
**Pull request by lindagcaba**
resolves #1244

### CODE REVIEW

File: opensrp-reveal/src/main/assets/ec_client_fields_zambia.json [hunk 1]
```diff
@@ -737,22 +737,6 @@
             "field": "obs.fieldCode",
             "concept": "sprayed"
           }
-        },
```
Comment (Rkareko) [inline opensrp-reveal/src/main/assets/ec_client_fields_zambia.json]: Why were these fields removed?

File: opensrp-reveal/src/main/assets/ec_client_fields_kenya.json [hunk 2]
```diff
@@ -0,0 +1,760 @@
...
+        {
+          "column_name": "sop",
+          "type": "Event",
+          "json_mapping": {
+            "field": "obs.fieldCode",
+            "concept": "cdd_name"
+          }
+        },
+        {
+          "column_name": "village_town",
```
Comment (Rkareko) [inline opensrp-reveal/src/main/assets/ec_client_fields_kenya.json:750]: Change this column name to `entity`

Comment (lindagcaba) [reply opensrp-reveal/src/main/assets/ec_client_fields_kenya.json]: updated.

**Comment by Rkareko** [comment]
@lindagcaba Update the PR with latest changes on master and work on fixing the drop in test coverage.

**Review by Rkareko — APPROVED** [review]

**Pull request merged**